nginx/server/sigopt.conf

```diff
@@ -1,6 +1,11 @@
 server {
   listen 4443 ssl;
 
+  # these options are required for minio
+  ignore_invalid_headers off;
```

Comment:
need to look a bit more into this option, required for signed aws access credentials iirc

Reply:
apparently this is just for the header names